features/cellular/framework/AT/ATHandler.cpp

```diff
@@ -150,7 +148,10 @@ FileHandle *ATHandler::get_file_handle()
 
 void ATHandler::set_file_handle(FileHandle *fh)
 {
+    _fh_sigio_set = false;
```

Comment:
Why to protect setting fileHandle, maybe this flag could be removed?

Reply:
good point, removed.